apps/script/checkEntryPoints.js

```diff
@@ -95,6 +95,7 @@ const SILENCED = [
   'tutorialExplorer',
   'weblab',
   'learn/index',
+  'teacherApplication'
```

Comment:
I thought you were not going to merge this? Looks like it is not needed anyway, since you created that new file in the sites directory.